src/api/AppApi.js

```diff
@@ -0,0 +1,254 @@
+/*
+ * @flow
+ */
+
+import Logger from '../utils/Logger';
+
+import {
+  APP_API
+} from '../constants/ApiNames';
+
+import {
+  APP_PATH,
+  BULK_PATH,
+  CONFIG_PATH,
+  INSTALL_PATH,
+  LOOKUP_PATH,
+  TYPE_PATH
+} from '../constants/ApiPaths';
+
+import {
+  getApiAxiosInstance
+} from '../utils/AxiosUtils';
+
+import {
+  isNonEmptyString
+} from '../utils/LangUtils';
+
+import {
+  isValidUuid,
+  isValidUuidArray
+} from '../utils/ValidationUtils';
+
+const LOG = new Logger('EntityDataModelApi');
+
+/**
+  * `GET /app/`
+  *
+  * Loads all apps.
+  *
+  * @static
+  * @memberof lattice.AppApi
+  * @return {Promise<Schema>} - a Promise that will resolve with a list of all apps
```

Comment:
Is `Schema` the right object being returned? I'd recommend just doing something like `Promise<Object>` or `Promise<Object[]>` until we figure out the right types.